client/my-sites/site-settings/form-writing.jsx

```diff
@@ -126,7 +126,7 @@ class SiteSettingsFormWriting extends Component {
 				}
 
 				{
-					this.renderSectionHeader( translate( 'Custom content types' ) )
+					this.renderSectionHeader( translate( 'Content types' ) )
```

Comment:
Nice tip @a8ci18n, but we're going with the lowercase one intentionally - it was a design decision a while ago. Should we change the original translations, @yoavf?